app/src/main/kotlin/de/digitalService/useID/util/AbTestManager.kt

```diff
@@ -0,0 +1,80 @@
+package de.digitalService.useID.util
+
+import androidx.compose.runtime.State
+import androidx.compose.runtime.mutableStateOf
+import de.digitalService.useID.analytics.IssueTrackerManagerType
+import de.digitalService.useID.analytics.TrackerManagerType
+import de.digitalService.useID.hilt.ConfigModule
+import io.getunleash.UnleashClient
+import io.getunleash.UnleashConfig
+import io.getunleash.UnleashContext
+import io.getunleash.polling.PollingModes
+import kotlinx.coroutines.suspendCancellableCoroutine
+import java.util.*
+import javax.inject.Inject
+import javax.inject.Named
+import javax.inject.Singleton
+import kotlin.coroutines.resume
+
+@Singleton
+class AbTestManager @Inject constructor(
+    @Named(ConfigModule.UNLEASH_API_URL) private val url: String,
+    @Named(ConfigModule.UNLEASH_API_KEY) private val apiKey: String,
+    private val trackerManager: TrackerManagerType,
+    private val issueTrackerManager: IssueTrackerManagerType
+) {
+
+    private lateinit var unleashClient: UnleashClient
+
+    private val _isSetupIntroTestVariant = mutableStateOf(false)
+    val isSetupIntroTestVariant: State<Boolean> = _isSetupIntroTestVariant
+
+    suspend fun initialise() = suspendCancellableCoroutine { continuation ->
+        val config = UnleashConfig.newBuilder()
+            .proxyUrl(url)
+            .clientKey(apiKey)
+            .pollingMode(PollingModes.autoPoll(Int.MAX_VALUE.toLong()))
+            .build()
+
+        val supportedAbTests = AbTest.values().joinToString(",") { it.testName }
+        val context = UnleashContext.newBuilder()
+            .appName("useid-android")
```

Comment:
We currently use `bundesIdent.iOS` and I would rather make naming consistent. Does `useid-android` makes more sense for you? If yes, I’ll rename it on iOS side to `useid-iOS`. But if you don’t care I would suggest going with `bundesIdent.Android` 🙂

Reply:
It was just the first thing that came to my mind and I didn't check it again, I personally have no preference for one over the other so I will just change it to `bundesIdent.Android` :)